test/actor_test.py

```diff
@@ -987,7 +987,7 @@ def check_intervals_non_overlapping(list_of_intervals):
     @ray.remote(num_gpus=1)
     def f1():
         t1 = time.monotonic()
-        time.sleep(0.1)
+        time.sleep(0.2)
```

Comment:
What was the issue here?

Reply:
While tasks were spread out between nodes, multiple tasks used the same workers.

Reply:
Sorry, I don't really know much about this test, but if this is sensitive to timing, will it pass okay on Travis?

Reply:
Yeah it does, on a non-travis machine the old number still worked but on travis we need double that.